modules/base/po/sk.popie

```diff
@@ -1,470 +1,470 @@
 msgid You didn't define any ACL groups yet.
-msgstr
+msgstr Zatiaľ si nedefinoval žiadne ACL skupiny.
 
 msgid No such group exists.
-msgstr
+msgstr Takáto skipina neexistuje.
 
 msgid Bad group name (regex `{regex}`)
-msgstr
+msgstr Chybné meno skupiny (regex `{regex}`)
 
 msgid ACL group can't be named like that (regex `{regex}`).
-msgstr
+msgstr ACL skupina takto nemôže byť pomenovaná (regex `{regex}`).
 
 msgid Bad parameter.
-msgstr
+msgstr Zlý parameter.
 
 msgid No such group.
-msgstr
+msgstr Takáto skupina neexistuje.
 
 msgid ACL group removed.
-msgstr
+msgstr ACL skupina bola odstránená.
 
 msgid **{count} rules** exported.
-msgstr
+msgstr Bolo exportovaných **{count} pravidiel**.
 
 msgid No such ACL rule.
-msgstr
+msgstr Takéto ACL pravidlo neexistuje.
 
 msgid Rule removed.
-msgstr
+msgstr Pravidlo odstránené.
 
 msgid I've removed {count} rules.
-msgstr
+msgstr {count} pravidiel bolo odstránených.
 
 msgid I'm expecting one JSON file.
-msgstr
+msgstr Očakávam jeden JSON súbor.
 
 msgid You have to upload the JSON file.
-msgstr
+msgstr Musíš nahrať JSON súbor.
 
 msgid Your JSON file contains errors.
-msgstr
+msgstr Tvoj JSON súbor obsahuje chyby.
 
 msgid I've imported **{new}** new and **{updated}** updated rules.
-msgstr
+msgstr Bolo importovaných **{new}** nových a **{updated}** aktualizovaných pravidiel.
 
 msgid Skipping:
-msgstr
+msgstr Preskakujem:
 
 msgid expected `bool`
-msgstr
+msgstr očakávam `bool`
 
 msgid expected `list`
-msgstr
+msgstr očakávam `list`
 
 msgid invalid group
-msgstr
+msgstr neplatná skupina
 
 msgid expected `int`
-msgstr
+msgstr očakávam `int`
 
 msgid duplicate
-msgstr
+msgstr duplikovať
```

Comment:
Zde je to ve formě podstatného jména ("duplikát"). Česky je to taky špatně, můžeš to opravit v obou souborech?